vpc-cli-reference.md

```diff
@@ -1451,18 +1441,11 @@ Create a load balancer listener with udp protocol.
 
 - **LOAD_BALANCER**: ID or name of the load balancer.
 - **--vpc**: ID or name of the VPC. It is only required to specify the unique resource by name inside this VPC.
-- **--protocol**: The listener protocol. Load balancers in the application family support tcp, http, and https. Load balancers in the network family support tcp and udp.
```

Comment:
Use asterisks (**tcp**) to make the protocol options **bold**.   I see this in multiple places.

Reply:
I do not have the proper permission to edit or create a PR in the repo.  Otherwise, Kumar approved.  All good.

Reply:
@austinmama granted the permission to repo for edit.  revalidated the values it was already with asterisks (** **) . Pls check and let me know if correction required.

@KumarGanesanIBM